Sming/Wiring/FakePgmSpace.cpp

```diff
@@ -1,4 +1,4 @@
-
+#include "WiringFrameworkDependencies.h"
```

Comment:
This was originally in the header. When including debug_progmem.h in external projects like third party thi would start including all sming header uselessly and generate all sorts of issues. This header is actually needed by the source file => we keep header file clean for including in external thirtparty projects